qiskit/schemas/backend_configuration_schema.json

```diff
@@ -255,7 +255,68 @@
                                 "type": "array",
                                 "minItems": 0,
                                 "description": "A list of available parametric pulse shapes",
-                                "items": {"type": "string"}}
+                                "items": {"type": "string"}},
+                            "qubit_channel_map": {
+                                "type": "array",
+                                "minItems": 1,
+                                "items": {"type": "array", "minItems": 0, "items": {"type": "string"}},
+                                "description": "An array of dimension n_qubits where each element <i> contains a list of channels mapping to qubit <i>."
+                                },
+                            "channel_config": {
+                                "type": "object",
+                                "minItems": 1,
+                                "patternProperties": {
+                                    "^d": {
+                                        "type": "object",
+                                        "properties": {
+                                            "type": {"type": "string"},
+                                            "purpose": {"type": "string"},
+                                            "operates": {
+                                                "type": "object",
+                                                "properties": {
+                                                    "qubits": {"type": "array", "minItems": 1, "items": {"type": "number"}}
```

Comment:
I think its fair for a channel to operate no qubits, so minItems can be removed I think.